explorer/client/src/components/search/Search.tsx

```diff
@@ -21,6 +26,24 @@ function getPlaceholderText(category: SearchCategory) {
     }
 }
 
+function isInputValid(category: SearchCategory, input: string): boolean {
+    if (overrideTypeChecks) return true;
```

Comment:
I want to get rid of this but it requires overhauling the test data, i think this works fine for now

Reply:
Yes - we can revisit this when we move from static JSON to that delivered by the API.